submissions/YuliiaHordiichuk/popup/index.html

```diff
@@ -0,0 +1,206 @@
+                                        <figcaption class="popup__text">Translate</figcaption>
+                                    </figure>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a class="popup__link" href="#">
+                                    <figure class="popup__fig">
+                                        <img class="popup__icon" src="icons/videos.png" alt="Videos">
+                                        <figcaption class="popup__text">Videos</figcaption>
+                                    </figure>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a class="popup__link" href="#">
+                                    <figure class="popup__fig">
+                                        <img class="popup__icon" src="icons/search.png" alt="Search">
+                                        <figcaption class="popup__text">Search</figcaption>
+                                    </figure>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a class="popup__link" href="#">
+                                    <figure class="popup__fig">
+                                        <img class="popup__icon" src="icons/photos.png" alt="Photos">
+                                        <figcaption class="popup__text">Photos</figcaption>
+                                    </figure>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a class="popup__link" href="#">
+                                    <figure class="popup__fig">
+                                        <img class="popup__icon" src="icons/apps.png" alt="Apps">
+                                        <figcaption class="popup__text">Apps</figcaption>
+                                    </figure>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a class="popup__link" href="#">
+                                    <figure class="popup__fig">
+                                        <img class="popup__icon" src="icons/documents.png" alt="Documents">
+                                        <figcaption class="popup__text">Documents</figcaption>
+                                    </figure>
+                                </a>
+                            </li>
+                        </ul>
+                    
+                        <input class='popup__checkbox' type="checkbox" id="popup-checkbox"> 
+                        <label class="popup__label" for="popup-checkbox" tabindex="0">More</label>
```

Comment:
Add margin-right for your More button or remove margin-left.
![Screenshot_114](https://user-images.githubusercontent.com/66283273/99678446-7eb7f700-2a83-11eb-97b7-d074bda2efdf.png)

Reply:
Done and done :)